docs/docs/guides/application-development.md

```diff
@@ -53,58 +48,129 @@ my-app/
 
 To run dependent services locally you can use the included `docker-compose.yaml` file. This file includes databases, DMSS service, and job service. 
 
-Start services by running:
+Start services in the background (-d detached mode) by running:
 
-```
-docker compose up
+```bash
+docker-compose pull
+docker compose up --build -d
```

Comment:
```suggestion
docker compose up
```
I don't think there is anything to build in this compose file. Every image is pulled.
Also, perhaps recommend not running detached? Most running locally will be interested in viewing the logs.